README.md

```diff
@@ -288,5 +288,43 @@ Here are some generated diagrams from the specification examples (with legend en
 
 2. [Send CloudEvent on Workflow completion Example](https://github.com/serverlessworkflow/specification/blob/master/examples/examples.md#send-cloudevent-on-workfow-completion-example)
 <p align="center">
-<img src="img/provisionorders.png" alt="Send Cloud Event on Workflow complation"/>
+<img src="img/provisionorders.png" alt="Send Cloud Event on Workflow completion"/>
 </p>
+
+#### Using Workflow Utils
+Workflow utility allows you to interpret the Serverless Workflow Model.
```

Comment:
Maybe change to something like:

"Workflow utils provide a number of useful methods for extracting information from workflow definitions."

Reply:
Have fixed this now.Thanks